backend/apps/volontulo/views/api.py

```diff
@@ -98,3 +102,21 @@ class OrganizationViewSet(viewsets.ModelViewSet):
     queryset = models.Organization.objects.all()
     serializer_class = serializers.OrganizationSerializer
     permission_classes = (permissions.OrganizationPermission,)
+
+    @detail_route(methods=['POST'], permission_classes=(AllowAny,))
+    # pylint: disable=invalid-name, no-self-use
```

Comment:
`@staticmethod` doesn't work here?

see: https://docs.python.org/3/library/functions.html#staticmethod

Reply:
It does. Initially I put @staticmethod decorator as a second one and in this case it isn't working so I changed that back into a  method. But it's ok, when decorator is put before @detail_route, what seems obvious.